cmd/invoke.go

```diff
@@ -39,15 +40,18 @@ var (
 
 var InvokeCmd = &cobra.Command{
 	Use:   "invoke",
-	Short: "Invoke a method on a given Dapr application. Supported platforms: Self-hosted",
+	Short: "Invoke a method on a given Dapr application. Supported platforms: Kubernetes and self-hosted",
 	Example: `
-# Invoke a sample method on target app with POST Verb
+# Invoke a sample method on target app with POST Verb in self-hosted mode
 dapr invoke --app-id target --method sample --data '{"key":"value"}
 
-# Invoke a sample method on target app with GET Verb
+# Invoke a sample method on target app with in Kubernetes
+dapr invoke -k --app-id target --method sample --data '{"key":"value"}
```

Comment:
`--data` has an unclosed single quote. Same for the above example in self-hosted mode.

Reply:
Thank you for review. I fixed it.